Add more tests for torch::arange

yf225 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #29633 Support C++ tensor advanced indexing
* **#29689 Add more tests for torch::arange**
* #29632 Use default dtype for torch::tensor(floating_point_values) and torch::tensor(empty braced-init-list) when dtype is not specified

Differential Revision: [D18465818](https://our.internmc.facebook.com/intern/diff/D18465818)

Commit:
Add more tests for torch::arange

kostmo commented:
## CircleCI build failures summary
As of commit c0f0cf4:
* **9/9** failures introduced in this PR
* **0/9** recognized as flaky



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=c0f0cf459bec3447bbebc3b01919fad72555874a).

---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

Commit:
Update on "Add more tests for torch::arange"

Add more tests for torch::arange

gh-metadata: pytorch pytorch 29689 gh/yf225/69/head

Commit:
Update on "Add more tests for torch::arange"

Add more tests for torch::arange

gh-metadata: pytorch pytorch 29689 gh/yf225/69/head

Commit:
Update on "Add more tests for torch::arange"

Add more tests for torch::arange

gh-metadata: pytorch pytorch 29689 gh/yf225/69/head
